content/en/blog/falco-talon-v0-2-0/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Introducing Falco Talon v0.1.0
+date: 2024-11-27
+author: Igor Eulalio
+slug: falco-talon-v0-2-0
+images:
+  - /blog/falco-talon-v0-1-0/images/falco-talon-featured.png
+tags: ["Talon", "Release"]
+---
+
+Today we announce the release of **Falco Talon 0.2.0** 🦅!
+
+Falco Talon 0.2.0 is a minor release that includes new actionners and outputs, add parameters to existing actionners, along one small fix on the check and print commands.
+
+## Features
+
+* Add `gcp:function` actionner
+* Add `gcp:gcs` output
+* Add `ignore_standalone_pods` parameter for `kubernetes:terminate` actionner
+* Allow to wait until the completion of `kubernetes:drain` by configuring `max_wait_period`
+* Use smaller image for the `kubernetes:tcpdump` actionner
+
+## Fixes
+
+* Allow to `check` and `print` commands to work without specifying a `config.yaml`
```

Comment:
Can you add more details, like snippets to use the actionners, a screenshot of GC Storage, ...